### ISSUE TITLE
SPAGOBI-630 - problem when I open two Jpivot OLAP documents in two different tabs

### ISSUE THREAD

**Issue by KnowageServiceAcc** [SpagoBILabs/SpagoBI#548]
## Details
|Key            |Value                               |
|---------------|------------------------------------|
|**Key**        |SPAGOBI-630|
|**Type**       |Bug|
|**Status**     |Open|
|**Priority**   |Major|
|**Resolution** ||
|**Affects**    |2.8.0|
|**Fix**        ||
|**Components** |SERVER/OLAP/JPivot|
|**Labels**     ||

## People
|Key         | Value |
|------------|-------|
|**Assignee**|Davide Zerbetto|
|**Reporter**|Alessio De Rossi [X]|

## Dates
|Key         | Value |
|------------|-------|
|**Created** |2011-07-07T14:48:21.000+0200|
|**Updated** |2014-01-07T15:26:32.000+0100|
|**Resolved**|*n.d.*|
 
## Description
If I open two different Jpivot OLAP documents in two different tabs and I start navigating them, the data of the last document became data of the first one.
In other words if I switch from a tab to the other and I do an oparation (e.g. drill through) I see the same document!

#### Filipus Febrianto Santoso Filipus Febrianto Santoso added a comment - 2012-09-13T10:51:44.922+0200
is this problem still do not have solution or patch file?
#### dan daq added a comment - 2014-01-07T15:26:32.421+0100
Hi everyone !

Is there any solution or workaround for this?

Thanks in advance
